spm-integration-test.sh

```diff
@@ -0,0 +1,90 @@
+UUID=$(uuidgen)
+BRANCH_NAME="$1-temp-${UUID}"
+SAMPLE_APP_TEMP_DIR="NativeAuthSampleAppTemp"
+current_date=$(date +"%Y-%m-%d %H:%M:%S")
+
+# Build framework
+
+git checkout -b "$BRANCH_NAME"
+rm -rf archive framework MSAL.zip
+
+xcodebuild -sdk iphonesimulator -configuration Release -workspace MSAL.xcworkspace -scheme "MSAL (iOS Framework)" archive SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES -archivePath archive/iOSSimulator CODE_SIGNING_ALLOWED=NO -quiet > build.log 2>&1
```

Comment:
can you print something to explain what each task is doing?

Reply:
Done